int32 id
string texture
string paint
int32[] corner_ids
int32[] feature_ids
